test/charms/test_smoke/reference-pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[project]
+name = "test-smoke"
+version = "0.1.0"
+description = "Add your description here"
```

Comment:
It feels like we could shove a description in here or remove this field (if I'm remembering correctly that description is optional).